src/data/schedule.ts

```diff
@@ -0,0 +1,65 @@
+import { HomeAssistant } from "../types";
+
+export const weekdays = [
+  "sunday",
```

Comment:
Can we start on Monday?

Reply:
Not for this. JS starts on Sunday

Reply:
Ideally, the calendar should start on Monday, just like shown here: https://github.com/home-assistant/core/pull/76566#issuecomment-1211673836

Reply:
Would be nice to keep it consistent

Reply:
@zsarnett right, for this array we track JS indexes, but that doesn't mean our UI can't start on Monday

Reply:
It starts based on your locale

Reply:
> Would be nice to keep it consistent

Consistent? Thats node red not HA